### PR TITLE
Enable Teams cucumber tests [SCI-4022]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4022](https://biosistemika.atlassian.net/browse/SCI-4022)

### What was done
Enabled tests, fixed scenario, add some configs

### CODE REVIEW

File: features/step_definitions/team_steps.rb [hunk 1]
```diff
@@ -32,3 +34,21 @@
...
+  end
+end
+
+Given('team settings page') do
+  visit teams_path
+  page.save_screenshot(full: true)
+end
+
+Then('I click leave team button') do
+  find("a[data-action='leave-user-team']").click
```
Comment (aignatov-bio) [inline features/step_definitions/team_steps.rb]: Better use text identifier and you can use it as general element:
```
Then("I click {string} button") do |button|
  find('.btn', text: button).click
end
```

And then call it 
```
And I click "Leave Team" button
```

File: features/step_definitions/team_steps.rb [hunk 2]
```diff
@@ -32,3 +34,21 @@
...
+    team = Team.find_by(name: row[:team])
+    team ||= FactoryBot.create(:team, name: row[:team])
+    user = FactoryBot.create(:user, row.slice('name', 'email'))
+    FactoryBot.create(:user_team, user: user, team: team, role: row[:role])
+  end
+end
+
+Given('team settings page') do
+  visit teams_path
+  page.save_screenshot(full: true)
```
Comment (aignatov-bio) [inline features/step_definitions/team_steps.rb]: remove it.
Also we can add support step:
```
Then('I make screenshot') do
  page.save_screenshot(full: true)
end
```
And you can easy add it between steps.

Comment (aignatov-bio) [reply features/step_definitions/team_steps.rb]: Also we can also add gem to automatically open this screenshot.
`page.save_and_open_screenshot(full: true)`

File: features/step_definitions/team_steps.rb (hunk 1 above)
Comment (urbanrotnik) [reply features/step_definitions/team_steps.rb]: The button contains 2 spans with an icon, is not text-only btn.

**Pull request merged**